docs/content/src/learn/tutorials/database-integrations/querying-mysql-database/src/querying_mysql_database/resources/sampleScript.sql

```diff
@@ -0,0 +1,27 @@
+#Sample script for Ballerina Integrator Inserting into MySQL Database 
```

Comment:
```suggestion
# Sample script for Ballerina Integrator Inserting into MySQL Database 
```

Reply:
And please check whether this sentence is grammatically correct

Reply:
Fixed